docs/setup/settings.asciidoc

```diff
@@ -504,6 +504,87 @@ deprecation warning at startup. This setting cannot end in a slash (`/`).
 proxy sitting in front of it. This determines whether HTTP compression may be used for responses, based on the request `Referer` header.
 This setting may not be used when <<server-compression, `server.compression.enabled`>> is set to `false`. *Default: `none`*
 
+
+a| [[server-securityResponseHeaders-strictTransportSecurity]]
+----
+server.securityResponseHeaders:
+  strictTransportSecurity:
+----
+| Controls whether the https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security[`Strict-Transport-Security`]
+header is used in all responses to the client from the {kib} server. Set this to `null` to disable it. *Default:* enabled
+
+a|
+----
+server.securityResponseHeaders:
+  strictTransportSecurity.maxAge:
+----
+| Specifies the `max-age` directive of the
+https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security[`Strict-Transport-Security`] response header. This is
+rounded down to the nearest second. *Default:* `'1Y'` (31536000 seconds)
+
+2+a|
+[TIP]
+============
+Use a string of `<count>[s\|m\|h\|d\|w\|M\|Y]` (e.g. '20m', '24h', '120d', '1Y').
+============
+
+a|
+----
+server.securityResponseHeaders:
+  strictTransportSecurity.includeSubDomains:
+----
+| Specifies the `includeSubDomains` directive of the
+https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security[`Strict-Transport-Security`] response header. If
+enabled, the directive is included in the header. *Default:* `false`
+
+a|
+----
+server.securityResponseHeaders:
+  strictTransportSecurity.preload:
```

Comment:
@elastic/kibana-docs  What's the best way of printing long settings like this? The table is too narrow to fit both the name and description. However, breaking it up like this makes it a bit confusing for users too.

Reply:
@thomheymann the table capabilities in Asciidoc are limited, and I agree that breaking up the setting names is confusing. These would be best displayed in a definition list instead of a table. I propose that we merge these changes, then I'll fix the formatting before 7.13 releases.